donut/modules/feedback/routes.py

```diff
@@ -15,7 +15,7 @@
 
 @blueprint.route('/feedback/<group>')
 def feedback(group):
-    if group not in Groups:
+    if group not in Groups or not auth_utils.is_caltech_user():
```

Comment:
I think `not (group in Groups and auth_utils.is_caltech_user())` is a bit clearer, but your call